falcon-tracer/conf/logging.ini

```diff
@@ -2,21 +2,27 @@
 keys=root
 
 [handlers]
-keys=consoleHandler
+keys=consoleHandler,fileHandler
 
 [formatters]
 keys=simpleFormatter
 
 [logger_root]
```

Comment:
I would duplicate this to create the new logger `logger_event` with its specific configurations.

Reply:
The new logger can be used in the `EventLogger` class.